plugins/adrv9002.c

```diff
@@ -1668,44 +1668,68 @@ static int profile_gen_config_set_live_device(struct adrv9002_config *cfg, gpoin
 	sprintf(str_value, "%d", cfg->radio_cfg.fdd);
 	gtk_combo_box_set_active_id(GTK_COMBO_BOX(gtk_builder_get_object(priv->builder, "cb_radio_duplex")), str_value);
 
-	// RX and TX
-	size_t ch_type;
-	char *ch_types[2] = {"rx", "tx"};
-	for(ch_type = 0; ch_type < ARRAY_SIZE(ch_types); ch_type++) {
-		for(chann = 0; chann < CHANNEL_COUNT; chann++) {
-			// channel_bandwidth_hz
-			sprintf(widget_str, "cb_%s_chan%d_bw", ch_types[ch_type], chann + 1);
-			sprintf(value, "%d", cfg->radio_cfg.rx_config[chann].channel_bandwidth_hz);
-			populate_combo_box(GTK_COMBO_BOX_TEXT(gtk_builder_get_object(priv->builder, widget_str)), NULL, 0, TRUE,
-					   value);
+	// TX
+	for(chann = 0; chann < CHANNEL_COUNT; chann++) {
```

Comment:
Note that there's already a `ADRV9002_NUM_CHANNELS`... But ideally you would use ARRAY_SIZE(). Also for tx, there's `n_tx` to accommodate for adrv9003 which only has 1 TX.

Hence, I'm not seeing the point of CHANNEL_COUNT